app.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+var pictureArray = ['bag.jpg', 'banana.jpg', 'bathroom.jpg', 'boots.jpg','breakfast.jpg','bubblegum.jpg','chair.jpg','cthulhu.jpg','dog-duck.jpg','dragon.jpg','pen.jpg','pet-sweep.jpg','scissors.jpg','shark.jpg','sweep.png','tauntaun.jpg','unicorn.jpg','usb.gif','water-can.jpg','wine-glass.jpg'];
+
+
+function Picture(name, path) {
+  this.name = name;
+  this.path = path;
+  this.numberOfTimesShowned = 0;
+  this.numberOfTimesClicked = 0;
+  this.generateRandonImage();
+}
+
+Picture.prototype.generateRandonImage = function () {
+  var generatedPicture = pictureArray[Math.floor(Math.random()* pictureArray.length)];
+  if(pictureArray.indexOf(generatedPicture == -1)) {
+    this.name = generatedPicture;
+    this.path = 'img/' + this.name;
+    // console.log(this.path)
+  } else {
+    this.generateRandonImage();
+  }
+  this.divIdCreate();
+  this.numberOfTimesShowned++;
+};
+
+Picture.prototype.divIdCreate = function () {
+  var displayPictureBoxes = document.getElementById('pictureArrays');
+  for(var i = 0; i < 1; i++){
+    var pictureDivCreate = document.createElement('div');
```

Comment:
Instead of creating a new image each time this function is called, have the images in your html and re-assign the src attribute each time.  This will prevent the images from forming an army of images on the html.